Gruntfile.js

```diff
@@ -110,27 +143,39 @@ module.exports = function(grunt) {
 			files: buildFiles.concat( [
 				'!wp-config.php',
 			] ).map( function( file ) {
-				if ( '!' === file.charAt( 0 ) ) {
-					return '!' + BUILD_DIR + file.substring( 1 );
-				}
-				return BUILD_DIR + file;
+				return setFilePath( BUILD_DIR, file );
 			} ),
-			css: [
-				WORKING_DIR + 'wp-admin/css/*.min.css',
-				WORKING_DIR + 'wp-admin/css/*-rtl*.css',
-				WORKING_DIR + 'wp-includes/css/*.min.css',
-				WORKING_DIR + 'wp-includes/css/*-rtl*.css',
-				WORKING_DIR + 'wp-admin/css/colors/**/*.css'
-			],
-			js: [
-				WORKING_DIR + 'wp-admin/js/',
-				WORKING_DIR + 'wp-includes/js/'
-			],
-			'webpack-assets': [
-				WORKING_DIR + 'wp-includes/assets/*',
-				WORKING_DIR + 'wp-includes/css/dist/',
-				'!' + WORKING_DIR + 'wp-includes/assets/script-loader-packages.min.php'
-			],
+
+			// Always clean built JS and CSS files from /src. Also from /build when building there.
+			// The build:css, build:js, etc. tasks may be run independently,
+			// and should clean all old files to avoid errors.
+			css: cssFiles.map( function( file ) {
+				return setFilePath( SOURCE_DIR, file );
+			} ).concat(
+				WORKING_DIR === BUILD_DIR
+				? cssFiles.map( function( file ) {
```

Comment:
@azaozz JavaScript coding standards shows error `Misleading line break before '?'; readers may interpret this as an expression boundary.`

Reply:
Yea, this was formatted like Prettier (as in Gutenberg) but seems there are some "double standards" for JS (was planning to look more there but got distracted) :) 
Will try to work around the inconsistency.

Reply:
Apparently no workaround... Prettier ternaries not allowed :)

Reply:
Awesome. Thanks for updating